app/assets/stylesheets/components/school_dashboard.scss

```diff
@@ -8,6 +8,12 @@
   display: flex;
   flex-direction: column;
   padding: 0 20px;
+  .Select-menu-outer {
```

Comment:
This could be a good place to add a comment explaining that since the Discipline dropdown menu is large, without increasing the max-height some of the options get cut off. We could also link to the react-select default CSS here since it looks like what you're doing here is bumping up the value of `@select-menu-max-height` from 200px to 1000px. 

+ https://github.com/JedWatson/react-select/blob/master/less/menu.less
+ https://github.com/JedWatson/react-select/blob/3cf1b7d18844ec309725239f03a22fb98485b8a7/less/select.less

Reply:
Also, taking a look on Internet Explorer VM, I'm still seeing scrolling behavior, which makes me think the CSS here isn't working the same way as in Chrome. There's also a scrollbar by default which I think would make it less likely that someone would miss the fact that they can scroll for more options. GIF from Internet Explorer 11 VM:

![menu](https://user-images.githubusercontent.com/3209501/40310711-4d6dfdd2-5cd3-11e8-93a9-57261d5fdcbe.gif)

Reply:
@alexsoble weird! It does show the full menu in my VM (gif above) but it doesn't look like a big deal b/c of the scroll bar. It was irritating on Chrome b/c the scrollbar isn't there until I try to scroll. 

Re linking the default css, I actually tried that, but rails objected to serving the css ([similar issue](https://stackoverflow.com/questions/7829480/no-route-matches-get-assets?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa)) I wasn't sure about changing our rails config just for that, but maybe I'm missing a better way.

Reply:
@alexsoble ah, gotha. Yes, I'll update the PR with some comments here.